landing_page/static/landing_page/js/index.js

```diff
@@ -38,14 +38,25 @@ async function searchMovies(event) {
         .then((data) => {
             moviesContainer = document.getElementById("searched_movies_container")
             moviesContainer.innerHTML = '';
-            data.movies.forEach(movie => {
+            searchbar = document.getElementById("searchbar")
+            searchbar.style.borderRadius = "12px 12px 0 0"
+
+            data.movies.forEach((movie, index) => {
                 // Create a new <div> element to display the movie details
                 const movieDiv = document.createElement('div');
                 movieDiv.classList.add('movie');
+                movieDiv.classList.add('searched_movie');
+                movieDiv.onclick = () => {
+                    window.location.href = window.location.origin + "/search/movies?query=" + encodeURIComponent(movie.name)
```

Comment:
I know it was mentioned earlier but yeah direct link to the movie info page would be nice

Reply:
I think it would be more intuitive to have clicking on the movie div lead directly to the movie page while pressing enter/clicking on the magnifying glass brings up the general search page

Reply:
I see what you're saying. I was thinking of the suggestions as more of an autocomplete but I think adding the details about the movie and a direct link works better

Reply:
This has been added